rpmalloc/rpmalloc.c

```diff
@@ -688,7 +688,7 @@ static int32_t _huge_pages_peak;
 //////
 
 //! Current thread heap
-#if (defined(__APPLE__) || defined(__HAIKU__)) && ENABLE_PRELOAD
+#if (defined(__APPLE__) || defined(__HAIKU__) || defined(__TINYC__)) && ENABLE_PRELOAD
```

Comment:
The __TINYC__ has to go outside the ENABLE_PRELOAD since it always has to use the pthread path

#if ((defined(__APPLE__) || defined(__HAIKU__)) && ENABLE_PRELOAD) || defined(__TINYC__)

Reply:
true. thanks.